Update adding-outside-collaborators-to-repositories-in-your-organizat… (#30443)
github/docs: content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md

Outside collaborators cannot be added to an {% data variables.enterprise.prodname_emu_enterprise %}. However, you can grant limited access to users outside your enterprise using the guest collaborator role. For more information, see "[AUTOTITLE](/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/roles-in-an-enterprise#guest-collaborators)."
{% endif %}

> Outside collaborators cannot be added to a team, only members of the organization can be.

Reviewer: ```suggestion
Outside collaborators cannot be added to a team, team membership is restricted to members of the organization.
```
Slight edit to the second half of the sentence.
